third_party/silabs/efr32_sdk.gni

```diff
@@ -54,6 +54,11 @@ declare_args() {
   # Argument to enable IPv4 for wifi
   # aligning to match chip_inet_config_enable_ipv4 default configuration
   chip_enable_wifi_ipv4 = false
+
+  # "wpa_mixed" to force enable WPA/WPA2 security on rs91x
+  # "wpa3_transition" to force enable WPA2/WPA3 security on rs91x
+  # "wpa3_only" to force enable WPA3 security on rs91x
+  rs91x_security_mode = "wpa3_transition"
```

Comment:
Setting by default to wpa3_transition can break it for all the users, since mostly all homes have wpa/wpa2 mode only